supervisor/shared/background_callback.c

```diff
@@ -67,6 +77,7 @@ void background_callback_run_all() {
     if (!callback_head) {
         return;
     }
+    last_background_tick = port_get_raw_ticks(NULL);
```

Comment:
Move this up into add_core and set it if `last_background_tick` is 0. We want to track how long the currently queued task has been waiting.

Set it to 0 below when callback_head is set to NULL.